### ISSUE TITLE
debug module missing with webpack -p option

### ISSUE THREAD

**Issue by brandongoode** [numical/script-ext-html-webpack-plugin#8]
```sh
webpack -p

module.js:327
    throw err;
    ^

Error: Cannot find module 'debug'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (.../node_modules/script-ext-html-webpack-plugin/index.js:3:15)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at Object.<anonymous> (.../webpack.config.js:4:34)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
```

**Comment by numical** [comment]
Oops - amazing this has not come up before.  Will update package.json.

**Comment by numical** [comment]
Released v1.4.2.  Please 'npm install' and check all is OK.
Recomment if not.
Thanks for the alert.
